### DISCUSSION TITLE
Repeatability of a test with the HiGHS solver on a MIP problem

### DISCUSSION THREAD

**Discussion by laxluco** [ERGO-Code/HiGHS#1462] [Q&A] [answered]
Hello everyone,
I have a question regarding the following topic:
I have a big MILP problem (hundreds of thousands of variables and constraints), I would like to know if when I solve the problem twice for the same input with the same parameter if I should expect a similar result. Should I expect the tree exploration in the branch & bound method to be similar (due to some internal seed) or is it possible that it will vary between tests?
My main concern is that it is possible that I may find a feasible solution in some tests and not in others.

**Comment by jajhall** [comment, marked as answer]
The HiGHS solvers are entirely deterministic, so runs are entirely repeatable.

That said, by setting the random_seed (https://ergo-code.github.io/HiGHS/dev/options/definitions/#random_seed) option, the MIP solver will run differently, but always the same for each seed value
